### PR TITLE
I Need The Code For My WebSite

### PR DESCRIPTION
**Pull request by CoolPerson879**
I Need JSS for my website
[My Website](https://rajscape.com)

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 flow-typed/
 **/node_modules/**
 packages/**/dist/
```
Comment (CoolPerson879) [inline .eslintignore:3]: ##Hacs Prevew

File: .github/ISSUE_TEMPLATE/---bug-report.md [hunk 2]
```diff
@@ -8,10 +8,10 @@ __Expected behavior:__
 A clear and concise description of what you expected to happen.
 
 __Describe the bug:__
-A clear and concise description of what happend instead of the expected behavior.
```
Comment (CoolPerson879) [inline .github/ISSUE_TEMPLATE/---bug-report.md:11]: Happened Not Happend

**Comment by kof** [comment]
Hey, that's an interesting PR, while I appreciate the effort, I can not merge it as it is, we need to split it up into various PRs and discuss them. I am happy to merge the docs typos right away, but not so sure about the rest.

Also, what does this mean? "I Need The Code For My WebSite"

**Comment by CoolPerson879** [comment]
Come ON!!!

File: packages/react-jss/src/index.d.ts [hunk 3]
```diff
@@ -59,13 +59,8 @@ interface CreateUseStylesOptions extends BaseOptions {
   name?: string
 }
 
-declare function createUseStyles<C extends string = string>(
-  styles: Styles<C>,
-  options?: CreateUseStylesOptions
```
Comment (CoolPerson879) [inline packages/react-jss/src/index.d.ts:63]: COME ON.

**Pull request closed without merging**